docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,127 @@
+---
+meta:
+  title: "DevTools"
+  parentTitle: "Guides"
+  description:
+    "Learn how to leverage Liveblocks DevTools to inspect and troubleshoot your
+    collaborative application."
+---
+
+## What are Liveblocks DevTools?
+
+Our [DevTools](/devtools) are a browser-based extension that integrates with
+Liveblocks and your local development environment. This allows you to easily
+troubleshoot, debug, and optimize your collaborative online experiences.
+
+<Banner title="Live demo video">
+
+If you want to see how this works in video, be sure to check out this
+[live demo](https://www.youtube.com/watch?v=-6Jxejp0iuw) on YouTube.
+
+</Banner>
+
+## How DevTools work
+
+Our DevTools communicate with Liveblocks servers in development mode and
+retrieve the current state of storage and presence (and history is coming
+soon!). The DevTools support one-way data syncs from the Liveblocks client and
+are compiled away in production, similar to other DevTools like React and Redux.
+
+<Banner title="Devtools Compatibility with legacy packages">
+
+Our DevTools are currently only available if you are on Liveblocks 0.19 or
+later. To view step-by-step instructions on how to upgrade your project to
+Liveblocks 0.19, check out our [upgrade guides](/guides/upgrading/).
+
+</Banner>
+
+## How to use DevTools
+
+### Downloading
+
+Our DevTools extension is available for all major web browsers:
+
+- [Chrome](https://chrome.google.com/webstore/detail/liveblocks-devtools/iiagocfmmhknpdalddkbiejnfmbmlffk)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/liveblocks-devtools/)
+- [Edge](https://microsoftedge.microsoft.com/addons/detail/liveblocks-devtools/hfecmmnilleegmjaegkjjklnjbgadikg)
+
+### Features and tips
+
+- You can quickly expand and collapse nested objects within storage by holding
+  the `⌥` key (option on Mac) and clicking on the object.
+- You can expand and collapse objects with one level of depth by using the arrow
+  keys or spacebar.
+- When a value in storage changes, the DevTools will highlight the value to
+  indicate that it has changed.
+- Storage values are searchable by key.
+- Hovering over a value in storage makes a tooltip appear. If clicked, it
+  displays the value of that key.
+
+### If DevTools shows an empty page
+
+If you open Devtools after a Liveblocks client has already been initialized, you
+may see an empty page. This is because the DevTools are only able to retrieve
+the current state of storage and presence. To fix this, you can refresh the page
+or open DevTools before initializing the Liveblocks client.
```

Comment:
As an engineer, I would typically already be working on a page locally before opening devtools.   I wouldn't expect to need to have the panel open first, and then reload my page for them to appear.  For that reason, I think this section should exist, but I think it could go in its own troubleshooting section instead.